src/identity/components/GlobalHeader/GlobalHeaderDropdown/index.tsx

```diff
@@ -40,6 +40,7 @@ export interface Props extends StandardFunctionProps {
   mainMenuHeaderText?: string
   mainMenuHeaderIcon?: IconFont
   mainMenuOptions: MainMenuItem[]
+  typeAheadOnly?: boolean
```

Comment:
`typeAheadOnly` is a bit arcane and requires some knowledge of the implementation of this component, and couples this property to the existing implementation. If we change the submenu to be something that isn't a `TypeAheadDropdown` component, this property will be hella confusing. I think we'd be better served by naming this in a way that describes what it's doing, rather than how it's doing it. For example `subMenuOnly` or `onlyRenderSubmenu`, etc.

Reply:
Ack, and makes sense. I'll go with `onlyRenderSubmenu`.